content/applications/sales/crm/pipeline/multi_sales_team.rst

```diff
@@ -2,24 +2,67 @@
 Multiple sales teams
 ====================
 
-Use the *Sales Teams* feature to manage several sales teams, departments, or channels, each with
-their own unique sales processes.
+The *Multi Teams* feature in Odoo's *CRM* app allows salespeople to be assigned to multiple sales
+teams. Multiple teams can be created and managed in one dashboard, each with their own pipeline for
+leads and opportunities.
 
+Enable multi teams
+==================
 
-Create a new sales team
+To allow salespeople to be assigned to more than one sales team, the :guilabel:`Multi Teams` setting
+needs to be enabled. First, navigate to :menuselection:`CRM app --> Configuration --> Settings`.
+Under the :guilabel:`CRM` section, tick the checkbox labeled :guilabel:`Multi Teams`. Then, click
+:guilabel:`Save` at the top-left of the page.
+
+.. image:: multi_sales_team/multi-team-setting.png
+   :align: center
+   :alt: The Multi Teams setting in the CRM app.
+
+.. _crm/assignment-rules:
+
+Enable assignment rules
```

Comment:
It's not clear to me how this section is exactly tied to the Multi Teams feature, and we already have a doc for the assignment rules. Can we more clearly make that connection? Otherwise if it's relevant in a tertiary way we can just do a **seealso**, no?